src/webmon_app/reporting/static/live_update.js

```diff
@@ -215,11 +215,12 @@ function MonitorPlot(element_id, nid, option){
 				   "</a>" +
 				   "<h3>Local Options</h3>" +
 				   "<form name='form_" + this.dialog_name + "'>" +
-				   "<div id='radio_" + this.dialog_name + "'>y scale: <input type='radio' class='check_scale' " +
-				   "id='scale_log_" + this.dialog_name + "' name='radio' " + log_radio + "onclick='Plots[" + nid + "].check_scale()'>" +
+				   "<b>Y-scale: </b>" +
+				   "<div id='radio_" + this.dialog_name + "'> <input type='radio' class='set_log_scale' " +
```

Comment:
I like the name change and setting the default version once things are setup. This is much clearer.